client/app/layouts/application.html

```diff
@@ -35,7 +35,7 @@
             <span class="badge" ng-show="vm.newNotifications">&#8226</span>
           </a>
         </li>
-        <li class="siteswitcher-item" ng-click="vm.closeMenus()">
+        <li>
```

Comment:
`closeMenus()` was added a while back because we had nav menus overlapping (site switcher, help and user menu)  before removing this function please verify this is no longer an issue (maybe post a gif?)